services/frontend-service/src/ui/ActionsCart/CheckoutDialog.test.tsx

```diff
@@ -17,25 +17,37 @@ Copyright 2021 freiheit.com*/
 import React from 'react';
 import { act, fireEvent, getByText, render } from '@testing-library/react';
 import { Spy } from 'spy4js';
-import { BatchAction, LockBehavior } from '../../api/api';
+import { BatchAction, Environment_Application_ArgoCD, LockBehavior } from '../../api/api';
 import { ActionsCartContext } from '../App';
 import { callbacks, CheckoutCart } from './CheckoutDialog';
+import { Context } from '../Api';
+import { makeApiMock } from './apiMock';
 
 const mock_useBatch = Spy.mock(callbacks, 'useBatch');
 
 const mock_setActions = Spy('setActions');
 const doActionsSpy = Spy('doActionsSpy');
 
 describe('Checkout Dialog', () => {
-    const getNode = (actions: BatchAction[]) => {
+    const getNode = (
+        actions: BatchAction[],
+        getOverviewState?: 'pending' | 'resolved' | 'rejected',
```

Comment:
```suggestion
        getOverviewState?: 'pending' | 'resolved' | 'rejected',
```
This should be a type.

Reply:
https://github.com/freiheit-com/kuberpult/pull/180/commits/14caa110faf013b941960a3354ef5e719cfa798f